benches/netbench/Cargo.toml

```diff
@@ -12,25 +12,42 @@ bytes = "1.1"
 clap = { version ="4.5", features = ["derive"] }
 core_affinity = "0.8"
 hdrhist = "0.5"
+hermit_bench_output = "0.1.0"
```

Comment:
Where's the source of this? This should be part of the Hermit org.

Reply:
https://github.com/hermit-os/hermit-bench-output
We need to add the proper links to the cargo project, though. Added an issue: https://github.com/hermit-os/hermit-bench-output/issues/3